SamuelSml8/hh_rickmorty_backend
# feat: Add PostgreSQL configuration and install dependencies necessary

### Description

This Pull Request introduces the PostgreSQL connection configuration in the `app.module.ts` file, adds an example `.env.example` file for environment variables, and installs dependencies in `package.json` to include `@nestjs/typeorm`, `pg`, and `typeorm`.

### Changes Made

- **app.module.ts**: TypeORM configuration to connect to PostgreSQL.
- env.example**: Example file for the required environment variables.
- **package.json**: Installation of dependencies for PostgreSQL integration.

### How to Test?

1. Make sure PostgreSQL is installed and running on your local machine.
2. Configure the environment variables in a `.env` file based on `.env.example`.
3. Run the application with `npm run start:dev` and verify that it connects correctly to the PostgreSQL database.
4. Check that the dependencies are installed correctly and that there are no errors in the application.

### Notes

- The entities are not yet defined in the project.
